gengokit/template/NAME-service/NAME-server/server_main.gotemplate

```diff
@@ -74,7 +74,7 @@ func main() {
 	// Wrap Endpoints with middlewares. See ../middlewares/endpoints.go
 	endpoints := svc.Endpoints{
 	{{range $i := .Service.Methods -}}
-		{{$i.Name}}Endpoint:    middlewares.WrapEndpoint({{ToLower $i.Name}}Endpoint),
+		{{$i.Name}}Endpoint:    {{ToLower $i.Name}}Endpoint,
```

Comment:
This template code is indented for readability of the template code. 

It all gets run through gofmt after the template is executed.

https://github.com/TuneLab/go-truss/blob/master/gengokit/generator/gen.go#L103

I think it improves the readability of the templates significantly. In pull  https://github.com/TuneLab/go-truss/pull/73 I did this indentation to all templates and thing were much more readable afterwards.